lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -1111,6 +1127,36 @@ void AtenLenTOp::getCanonicalizationPatterns(RewritePatternSet &patterns,
   });
 }
 
+//===----------------------------------------------------------------------===//
+// AtenMulLeftTOp
+//===----------------------------------------------------------------------===//
+
+void AtenMulLeftTOp::getCanonicalizationPatterns(RewritePatternSet &patterns,
+                                                 MLIRContext *context) {
+  // `[3] * 5` -> `[3,3,3,3,3]`, if it is not mutated.
```

Comment:
It might be better to have an example like `[1,2]*2 = [1,2,1,2]` in this comment.